Client/Application/WebServer.swift

```diff
@@ -86,6 +86,14 @@ class WebServer {
         if components.host == "localhost" && components.scheme == "http" {
             components.port = Int(WebServer.sharedInstance.server.port)
         }
+        
+        // If the request was privileged
+        // Update its privilege token
+        if let url = components.url,
+           PrivilegedRequest.isWebServerRequest(url: url),
+           components.valueForQuery(PrivilegedRequest.key) != nil {
```

Comment:
This will leak the token to any request of the form `http://localhost:port/random.html?<key>=any_value`

Reply:
Lucky me. This code is no longer needed as the token is handled properly in `SessionRestore.html`. I am removing it now.